### PR TITLE
Remove all binding by arrow methods

### PR DESCRIPTION
**Pull request by aljesusg**
Removed all binding methods by arrow sintax

### CODE REVIEW

File: src/components/Badge/Badge.tsx [hunk 1]
```diff
@@ -23,15 +23,14 @@ class Badge extends React.Component<BadgeProps, BadgeState> {
...
-    this.updateTextLength = this.updateTextLength.bind(this);
   }
   componentDidMount() {
     this.updateTextLength();
   }
   componentDidUpdate() {
     this.updateTextLength();
   }
-  updateTextLength() {
+  updateTextLength = () => {
```
Comment (lucasponce) [inline src/components/Badge/Badge.tsx]: This is interesting, so, declaring the function object then simplify the binding declaration ?
I'm just learning, the preliminar examples I saw in the react site used all the bind(this) syntax.

Comment (jotak) [reply src/components/Badge/Badge.tsx]: Yes it's a subtlety in typescript, this was discussed here https://github.com/kiali/kiali-ui/pull/179 and I think there was also a topic on kiali-dev.

Comment (lucasponce) [reply src/components/Badge/Badge.tsx]: Nice. +1

**Comment by aljesusg** [comment]
Yes @lucasponce I was working in another place and  I realized that there are a lot of binding methods.  Thanks @jotak for merge.

**Pull request merged**